presto-docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -23,47 +22,363 @@ connection properties as appropriate for your setup:
+
+    - ``IGNORE``
+  * - ``oracle.number.default-scale``
+    - ``number_default_scale``               
+    - Default Presto ``DECIMAL`` scale for Oracle ``NUMBER`` (without precision 
+      and scale) date type. When not set then such column is treated as not 
+      supported.
+    - not set
+  * - ``oracle.number.rounding-mode``
+    - ``number_rounding_mode``               
+    - Rounding mode for the Oracle ``NUMBER`` data type. This is useful when 
+      Oracle ``NUMBER`` data type specifies higher scale than is supported in 
+      Presto. Possible values are:
+
+      - ``UNNECESSARY`` - Rounding mode to assert that the 
+        requested operation has an exact result, 
+        hence no rounding is necessary.
+      - ``CEILING`` - Rounding mode to round towards
+        positive infinity.
+      - ``FLOOR`` - Rounding mode to round towards negative
+        infinity.
+      - ``HALF_DOWN`` - Rounding mode to round towards
+        ``nearest neighbor`` unless both neighbors are
+        equidistant, in which case rounding down is used.
+      - ``HALF_EVEN`` - Rounding mode to round towards the
+        ``nearest neighbor`` unless both neighbors are
+        in which case rounding towards the nearest neighbor is
+        performed.
+      - ``HALF_UP`` - Rounding mode to round towards
+        ``nearest neighbor`` unless both neighbors are
+        equidistant, in which case rounding up is used
+      - ``UP`` - Rounding mode to round towards zero.
+      - ``DOWN`` - Rounding mode to round towards zero.
+
+    - ``UNNECESSARY``
+
+Oracle to Presto type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Presto supports selecting Oracle database types. This table shows the Oracle to
+Presto data type mapping:
+
+
+.. list-table:: Oracle to Presto type mapping
+  :widths: 20, 20, 60
+  :header-rows: 1
+
+  * - Oracle database type
+    - Presto type
+    - Notes
+  * - ``NUMBER(p, s)``
+    - ``DECIMAL(p, s)``
+    -  See :ref:`number mapping`
+  * - ``NUMBER(p)``
+    - ``DECIMAL(p, 0)``
+    - See :ref:`number mapping`
+  * - ``FLOAT[(p)]``
+    - ``DOUBLE``
+    -
+  * - ``BINARY_FLOAT``
+    - ``REAL``
+    -
+  * - ``BINARY_DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``VARCHAR2(n CHAR)``
+    - ``VARCHAR(n)``
+    -
+  * - ``VARCHAR2(n BYTE)``
+    - ``VARCHAR(n)``
+    -
+  * - ``NVARCHAR2(n)``
+    - ``VARCHAR(n)``
+    -
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``NCHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``CLOB``
+    - ``VARCHAR``
+    -
+  * - ``NCLOB``
+    - ``VARCHAR``
+    -
+  * - ``RAW(n)``
+    - ``VARBINARY``
+    -
+  * - ``BLOB``
+    - ``VARBINARY``
+    -
+  * - ``DATE``
+    - ``TIMESTAMP``
+    - See :ref:`datetime mapping`
+  * - ``TIMESTAMP(p)``
+    - ``TIMESTAMP``
+    - See :ref:`datetime mapping`
+  * - ``TIMESTAMP(p) WITH TIME ZONE``
+    - ``TIMESTAMP WITH TIME ZONE``
+    - See :ref:`datetime mapping`
+
+If an Oracle table contains a type that's not listed in the above table, an
```

Comment:
This one is rendered weirdly with extra indentatinon. Not sure how to fix that.

![image](https://user-images.githubusercontent.com/890111/91697416-61a11580-eb71-11ea-967e-3ec110aab7a9.png)

Reply:
I'm not sure how to fix this but I'm working on it. It's structured the same way the other lists in the document are structured.

Reply:
The problem that it is hard to follow (at least to me) that it says about different possible outcomes, depending on value of configuration property `unsupported-type.handling-strategy`.
Also I have just noticed that each option is marked with "(default configuration)" :).
Also it should be `unsupported-type.handling` (without `-strategy`); The version with `-strategy` would work too, but it is legacy and will be removed eventually.

I would expect something like:

```
You can specify Presto behaviour for situation when type not listed above is used in Oracle table by setting `unsupported-type.handling-strategy` configuration property.

If ``unsupported-type.handling`` is set to ``FAIL``  querying table with unsupported table will fail.

If ``unsupported-type.handling`` is set to ``IGNORE`` (default configuration) columns with unsupported types will not be exposed in Presto.

If ``unsupported-type.handling`` is set to ``CONVERT_TO_VARCHAR`` column will be exposed as unbounded VARCHAR
```

Of course please make it more English :)